migration/sql-files/018-work-item-description-update-search-index.sql

```diff
@@ -0,0 +1,33 @@
+-- migrate work items description
+update work_items set fields=jsonb_set(fields, '{system.description}', 
+  jsonb_build_object('content', fields->>'system.description', 'markup', 'plain'))
+  where fields->>'system.description' is not null;
+
+
+-- update support for Full Text Search Vector on work item description
+DROP TRIGGER IF EXISTS upd_tsvector ON work_items;
+DROP FUNCTION IF EXISTS workitem_tsv_trigger() CASCADE;
+DROP INDEX IF EXISTS fulltext_search_index;
+CREATE INDEX fulltext_search_index ON work_items USING GIN (tsv);
+
+-- update the 'tsv' column with the text value of the existing 'content' 
+-- element in the 'system.description' JSON document
+UPDATE work_items SET tsv =
+    setweight(to_tsvector('english', id::text),'A') ||
+    setweight(to_tsvector('english', coalesce(fields->>'system.title','')),'B') ||
+    setweight(to_tsvector('english', coalesce(fields#>>'{system.description, content}','')),'C');
```

Comment:
Yes, as explained on https://www.postgresql.org/docs/9.3/static/functions-json.html, the `#>>` gets a JSON object at specified path as returns the value as `text`